src/finance/store/actions.ts

```diff
@@ -21,4 +23,56 @@ export const actions: ActionTree<TariffStateInterface, RootStateInterface> & Act
       console.log(error);
     }
   },
+
+  [TariffActionEnum.SET_TARIFF_LIST]: async ({ commit }, payload) => {
+    try {
+      // TODO when endpoints will exist for finance section, create request to backend -> GET axios
+      // const response = await HTTP.get(``);
+      // const tariffList: Array<TariffModel> = response.data.map((el: TariffDTOModel) => new TariffModel(el));
```

Comment:
Here some comments to remove

Reply:
I'm going to uncomment this lines and add URL when endpoints will exist.